xml/System/ReadOnlySpan`1.xml

```diff
@@ -483,11 +531,14 @@
         <Parameter Name="right" Type="System.ReadOnlySpan&lt;T&gt;" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="left">The first span to compare.</param>
+        <param name="right">The second span to compare.</param>
+        <summary>Returns a value that indicates whether two <see cref="T:System.ReadOnlySpan`1" /> instances are equal.</summary>
+        <returns>
+          <see langword="true" /> if the two <see cref="T:System.ReadOnlySpan`1" /> instances are equal; otherwise, <see langword="false" /></returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[This tests if two <xref:System.ReadOnlySpan`1> instances point to the same starting memory location, and have the same <xref:System.ReadOnlySpan`1.Length> values. This does not compare the contents of two <xref:System.ReadOnlySpan`1> instances.]]></format>
```

Comment:
I think the change from `value types` to `instances` here is great. Can we apply the same change to the other areas, like op_InEquality:
https://github.com/dotnet/dotnet-api-docs/pull/358/files#diff-6f643f9c5e5dcd33509a5c8e52ef06aeR634

And here (ToString):
https://github.com/dotnet/dotnet-api-docs/pull/358/files#diff-6f643f9c5e5dcd33509a5c8e52ef06aeR752